docs/content/wiring.md

```diff
@@ -25,6 +25,35 @@ You can also provide any other attributes, as specified by the CNAB [parameters]
   default: "wordpress"
 ```
 
+### File Parameters
+
+Porter also enables the use of file parameters in a bundle.
+
+For instance, a bundle might declare a parameter `mytar` of type `file`, to exist at `/root/mytar` in the runtime container:
```

Comment:
I think we should avoid saying "container" when we are referring to the bundle's execution environment at runtime. It's probably not just "bundle runtime" since that refers to what executes the bundle (in spec talk).

I'm not 100% sure if we have been consistent in our docs, or if there is an official term from the CNAB world. Just that avoiding conflating containers and the bundle runtime would be a good idea.

If you have an idea for what to call it, go for it. I know that when I'm talking about it I've said "bundle runtime environment" and "execution environment" and all sorts of things. 🤷 It would be nice to know what's the right term! 😅